### ISSUE TITLE
Enhancement: Move over to the Steam Input glyphs

### ISSUE THREAD

**Issue by AWildErin** [ChaosInitiative/Portal-2-Community-Edition#955]
~~No template for enhancement so I don't have much of a format~~

We should probably migrate away from the ugly steam controller glyphs currently used in P2CE. Steam Input brought with it a bunch of nice sleek-looking icons and I feel we could make use of them. Luckily, the API has a function to get the correct glyph it needs to use directly from the steam folder, so we don't need to ship any new icons or anything.

Steam Input Icons:
They come in 3 different styles and sizes in png and SVG form, for generally all supported Steam Input controllers (PS, Switch, Xbox, Steam Controller and Steam Deck)
* Light:
![shared_color_button_a_sm](https://user-images.githubusercontent.com/28122025/180203175-e07e3ed1-ba32-4009-b8c9-39f054082750.png)
![shared_color_outlined_button_a_sm](https://user-images.githubusercontent.com/28122025/180203179-6b76745c-6a08-4da0-987f-3c1a2bfef387.png)

* Dark:
![shared_color_button_a_sm](https://user-images.githubusercontent.com/28122025/180202984-b63a0450-eca2-4d56-8ee6-1b359c5e40bf.png)
![shared_color_outlined_button_a_sm](https://user-images.githubusercontent.com/28122025/180203145-9f02bcbb-5fb0-4722-a0b1-0d7d798e5022.png)

* Knockout:
![shared_color_button_a_sm](https://user-images.githubusercontent.com/28122025/180203050-856a8a16-7d8e-4900-895f-709ff9d01216.png)
![shared_color_outlined_button_a_sm](https://user-images.githubusercontent.com/28122025/180203044-a1035e79-bd37-446d-bcb2-7fcd4dbd5a6a.png)

In contrast, here is the current icon we use for the `A` button:
![image](https://user-images.githubusercontent.com/28122025/180203463-8516c8fb-1aaf-46a9-b970-49cf42ca8607.png)

**Comment by Luke18033** [comment]
Portal 2 already uses these. It'd probably just make sense to implement this whenever Panorama controller support is worked on

**Comment by vrad-exe** [comment]
I think these are used now?

**Comment by iamsirenstorm** [comment]
Yeah, we wired in Steam Input Glyphs to Panorama Panels a while ago, and those are currently in use. Probably won't bother with Legacy VGUI. Closing as complete, thanks for the bump.
